src/plots/cartesian/set_convert.js

```diff
@@ -243,6 +351,6 @@ module.exports = function setConvert(ax) {
     ax._max = [];
 
     // and for bar charts and box plots: reset forced minimum tick spacing
-    ax._minDtick = null;
-    ax._forceTick0 = null;
+    delete ax._minDtick;
+    delete ax._forceTick0;
```

Comment:
If we do that, these will still get iterated over and `delete`d in `nestedProperty: pruneContainers` - which is why I had to make this change in the first place because it deletes `null` too - so in this case it would actually marginally slow things down. There may be some broader cleanup for us to do to avoid that whole mess but that's for another time...